lib/report.js

```diff
@@ -112,7 +100,17 @@ class ConciseReporter extends Reporter {
   listFailure(test, res, message) {
     if (res.stack) res.stack = renderStack(res.stack);
     this.printAssertErrorSeparator();
-    console.log(list(res, message, ['success']) + '\n');
+    let s = message;
+    ['type', 'message', 'expected', 'actual', 'stack'].forEach(key => {
+      let value = res[key];
+      if (key === 'actual' || key === 'expected') {
+        let type = typeof value;
+        if (type === 'object') type = value.constructor.name;
```

Comment:
There is also a `null`-check missing here, if the `value` is `null`, this line throws.